packages/eslint-plugin/tests/rules/array-type.test.ts

```diff
@@ -1378,7 +1382,7 @@ interface ArrayClass<T> {
           messageId: 'errorStringArraySimple',
           data: { className: 'Array', readonlyPrefix: '', type: 'T' },
           line: 3,
-          column: 8,
+          column: 12,
         },
```

Comment:
[Tests] Could you revert these changed `column`s? These changes don't seem to be related to this PR.